autodocodec/package.yaml

```diff
@@ -20,6 +20,7 @@ library:
   - aeson
   - bytestring
   - containers
+  - dlist
```

Comment:
Do we already (transitively) depend on `dlist`?

Reply:
I'm not sure. But I presume we need to explicitly depend on it to create the instances.

`dlist` is pretty lightweight, the only thing it depends on outside of what is distributed with GHC is `deepseq`. And `deepseq` itself depends only on `base`. Which is why I didn't create a new library/orphans.

Reply:
Ah yes `aeson` depends on `dlist so then this is fine.